README.md

```diff
@@ -67,16 +67,35 @@ You can schedule tasks to a given cpu with:
 taskset --cpu-list 5 shasum /dev/zero
 ```
 
+While this works properly for programs which spawn a single thread, this fails to work for a program launching multiple threads. This has been documented here [isolcpus_taskset_bug](https://bugzilla.kernel.org/show_bug.cgi?id=116701). As per the Linux kernel devs, `isolcpus` completely removes the cores from the scheduler. Thus the ideal way to work around this is by changing the scheduler for the running process from `SCHED_OTHER` (default) to something else. This can be done via the `chrt` command [chrt_ref](http://man7.org/linux/man-pages/man1/chrt.1.html).
+
+```
+sudo taskset --cpu-list 2-16 chrt -r 1 <name-of-executable>
```

Comment:
I know it is quite icky, that we need sudo to run multithreaded programs. There are 2 ways to work around it - 

1. Use `cpuset` instead of `isolcpus`. I have to experiment a bit more with `cpuset` to see how it performs compared to `isolcpus`.
2. Use it with cgroups, to allow use to run sudo and change the scheduler. Again, not very sure of this approach.